### PR TITLE
Template changes for Scaffolding

### PR DESCRIPTION
**Pull request by prafullbhosale**
- Update the scripts section to match Templates
- Fix Edit view template to always add a hidden input for the primary key
- Fix select tag helper for foreign keys in the Create and Edit view templates

cc @rustd @sayedihashimi @abpiskunov

### CODE REVIEW

**Comment by prafullbhosale** [comment]
cc @balachir @phenning

**Comment by prafullbhosale** [comment]
cc @Eilon, @DamianEdwards  (for RC2 ask mode approval).

**Comment by prafullbhosale** [comment]
Tagging aspnet/Release/issues/4

File: src/Microsoft.Extensions.CodeGenerators.Mvc/Templates/ViewGenerator/Create.cshtml [hunk 1]
```diff
@@ -51,18 +51,17 @@
...
             @:<label asp-for="@property.PropertyName" class="col-md-2 control-label"></label>
             @:<div class="col-md-10">
-                @:<select asp-for="@property.PropertyName" class ="form-control"></select>
+                @:<select asp-for="@property.PropertyName" class ="form-control" asp-items="@property.PropertyName"></select>
             @:</div>
         @:</div>
                 continue;
             }
-
+
```
Comment (phenning) [inline src/Microsoft.Extensions.CodeGenerators.Mvc/Templates/ViewGenerator/Create.cshtml]: Nit: trailing whitespace

File: src/Microsoft.Extensions.CodeGenerators.Mvc/Templates/ViewGenerator/Edit.cshtml [hunk 2]
```diff
@@ -43,30 +43,31 @@
...
-            {
-        @:<input type="hidden" asp-for="@property.PropertyName" />
-                continue;
-            }
             if (property.IsReadOnly)
             {
                 continue;
             }
-
+
```
Comment (phenning) [inline src/Microsoft.Extensions.CodeGenerators.Mvc/Templates/ViewGenerator/Edit.cshtml]: whitespace

File: src/Microsoft.Extensions.CodeGenerators.Mvc/Templates/ViewGenerator/Edit.cshtml [hunk 3]
```diff
@@ -43,30 +43,31 @@
...
             @:<div class="col-md-10">
-                @:<select asp-for="@property.PropertyName" class="form-control" />
+                @:<select asp-for="@property.PropertyName" class="form-control" asp-items="@property.PropertyName"></select>
                 @:<span asp-validation-for="@property.PropertyName" class="text-danger" />
             @:</div>
         @:</div>
                 continue;
             }
-
+
```
Comment (phenning) [inline src/Microsoft.Extensions.CodeGenerators.Mvc/Templates/ViewGenerator/Edit.cshtml]: whitespace

**Comment by Eilon** [comment]
Why does the Edit view need a hidden input with the primary key? Isn't the primary key already in the URL?

**Comment by Eilon** [comment]
Or is it for cases when the ID isn't called `id`?

**Comment by prafullbhosale** [comment]
Yes. Its also needed for the form submit.

**Comment by Eilon** [comment]
Ok :shipit: approved for RC2.

**Pull request closed without merging**